### PR TITLE
[SD WM] Move the OCD-ID fallback into user-works

### PR DESCRIPTION
**Pull request by sduckett**
The second half of [turbovote-web pull request](https://github.com/democracyworks/turbovote-web/pull/91) and should be deployed at the same time.

- Blocked by [a bifrost pull request](https://github.com/democracyworks/bifrost/pull/3) and a [user-http-api pull request](https://github.com/democracyworks/user-http-api/pull/12).
- [Pivotal Card](https://www.pivotaltracker.com/story/show/114324515)

### CODE REVIEW

File: src/user_works/handlers.clj [hunk 1]
```diff
@@ -43,9 +52,9 @@
       (if-not (= :timeout response)
         (if-let [ocd-divisions (:ocd-divisions response)]
```
Comment (7maples) [inline src/user_works/handlers.clj]: Are we confident that we never get an empty set out of the response?

File: src/user_works/handlers.clj [hunk 2]
```diff
@@ -33,6 +33,15 @@
                  :server (str "Unknown server error: "
                               (.getMessage runtime-exception))}}))))
 
+(defn fallback-ocd-id [state]
+  (let [country-ocd-id "ocd-division/country:us"
```
Comment (7maples) [inline src/user_works/handlers.clj]: Doc string s'il vous plait

File: src/user_works/handlers.clj (hunk 1 above)
Comment (cap10morgan) [inline src/user_works/handlers.clj]: Good point. I don't think I've seen an empty set come back, but it certainly could. We should fall back to country and state then too!

File: src/user_works/handlers.clj (hunk 2 above)
Comment (sduckett) [inline src/user_works/handlers.clj]: certainly.

File: src/user_works/handlers.clj (hunk 1 above)
Comment (sduckett) [inline src/user_works/handlers.clj]: I'll have a test.

**Comment by sduckett** [comment]
I've addressed your comments, @7maples.

**Comment by 7maples** [comment]
LGTM

**Pull request merged**